libs/databricks/tests/unit_tests/test_vectorstore.py

```diff
@@ -133,12 +133,12 @@ def embed_query(self, text: str) -> List[float]:
 
 @pytest.fixture(autouse=True)
 def mock_vs_client() -> Generator:
-    def _get_index(endpoint: str, index_name: str) -> MagicMock:
+    def _get_index(
+        endpoint_name: Optional[str] = None,
+        index_name: str = None,  # type: ignore
```

Comment:
```suggestion
        index_name: Optional[str] = None,
```

can we add Optional?

Reply:
If we do that, L143 complains that the dictionary key should be non-null. Practically we set `index_name` always, but since `endpoint_name` is optional we have to add some default here (and the [original VS client API](https://api-docs.databricks.com/python/vector-search/databricks.vector_search.html#databricks.vector_search.client.VectorSearchClient.get_index) has the arguments in this order).

I'm actually thinking of disabling mypy in test code. It raises so many false alarms especially when we add integration tests (because many module miss type stubs).

Reply:
got it :)